packages/@okta/i18n/src/properties/login.properties

```diff
@@ -645,6 +645,7 @@ password.complexity.minAgeHours = At least {0} hour(s) must have elapsed since y
 # {0} is a number
 password.complexity.minAgeDays = At least {0} day(s) must have elapsed since you last changed your password.
 password.reset.verification = Verify with one of the following factors to reset your password.
+password.reset.revokeSessions = Sign me out of all other devices.
```

Comment:
I would touch based with a designer about this messaging. The sessions being revoked may or may not be associated with a separate device

Reply:
thanks for the callout, I talked to Maggie and Vishwa and confirmed this is the text we are going with